samples/core/parameterized_tfx_oss/taxi_pipeline_notebook.ipynb

```diff
@@ -239,7 +239,7 @@
     "      trainer, model_analyzer, model_validator, pusher\n",
     "  ],\n",
     "  enable_cache=True,\n",
-    "  beam_pipeline_args=['--direct_num_workers=%d' % 4],\n",
+    "  beam_pipeline_args=['--direct_num_workers=%d' % 1],\n",
```

Comment:
would prefer you set it as 0 as it will be auto selected based on available cpus.
and maybe refine the title